subcommand.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+)
+
+type SubCommand struct {
+	cmd      *exec.Cmd
+	PipeRead *os.File
+}
+
+type Stream int
+
+const (
+	StreamStdout Stream = 1 << iota
+	StreamStderr
+)
+
+func NewSubCommand(args []string, stream Stream) SubCommand {
```

Comment:
s/Stream/int, this doesn't make this more typesafe, someone can still call this with `42`